docs/blog/2019-03-27-security-concerns-unique-to-static-sites/index.md

```diff
@@ -0,0 +1,128 @@
+
+- **server-side site** - Any site that renders dynamic content on the server, at time of request -- whether using node.js, PHP, Python, or any other method -- before delivering it to the user's browser.
+
+- **client-side site** - Gatsby only generates static sites. Single Page Apps (SPAs) are different but any security issue affecting a pure static site will also affect an SPA. Collectively both methods deliver all needed assets(HTML/CSS/JS) to the client for site rendering.
+
+- **attack vector** - A resource through which to hack a site. Just because an attack vector exists doesn't mean you site is vulnerable to that attack. A server is an attack vector. But, the server requires a vulnerability for an attacker to exploit(use) the server as an attack vector.
+
+## Shared Security Concerns
+
+No matter how you deploy your site, how little javascript you use, whether it is sever-side or client-side: your site has attack vectors.
+
+When it comes to static sites, if you are going to get hacked, it is going to be at the infastructure level (server/CDN, DNS, CI, etc.) or network level([MitM](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)). Thanks to the cloud, most of this work is done for you and you do not have to secure these layers. As long as you are using two-factor authentication(2FA) and a strong password. You are all using 2FA (when possible) and strong passwords, right!? Good.
+
+Other possible attacks relate to using `iframes` or not removing referrer tags on external links. Thankfully, fixing these is a pretty low bar. This [article](https://blog.sqreen.com/static-websites-security/) covers all these security issues well and includes solutions and deeper discussion.
+
+## Unique Security Concerns
+
+When you ship **everything** to the browser, the user can access **everything**. No amount of minification or obfuscation can hide the code and content you're shipping. If a user knows how to use their browsers dev tools, the user can get that content. This makes showing dynamic or private content on client-side sites seemingly impossible. This is why server-side rendering of sites was invented.
+
+In the server-side world the solution is to...well there is not really a problem to begin with because all the secrets and private content is secured on a server. The client browser never needs to directly access them. The server fetches content from the API or database, and sends back only the content the client is authorized to access.
+
+In the client-side world, how do you secure them, If all the content and code is being shipped to the client browser? The "[JAMstack](https://jamstack.org/)". Hopefully you're all fairly familiar with this concept given you're building "JAMstack" sites using gatsby. For the uninitiated: You use HTML/CSS ([M]arkup) to build your site, [J]avascript to make it dynamic, and [A]PIs (accessed by javascript) to provide content and features. All this being done from the client-side.
```

Comment:
```suggestion
How do you secure secrets in the client-side world, if all the content and code is being shipped to the client browser? An answer can be found with the "[JAMstack](https://jamstack.org/)". Hopefully you're all fairly familiar with this concept given you're building "JAMstack" sites using Gatsby. For the uninitiated: you use HTML/CSS ([M]arkup) to build your site, [J]avaScript to make it dynamic, and [A]PIs (accessed by JavaScript) to provide content and features. All of this is done from the client-side.
```

Reply:
Maybe reorder to fit the order of: J-A-M